jsx/Markdown.js

```diff
@@ -40,6 +40,6 @@ Markdown.propTypes = {
 let RMarkdown = React.createFactory(Markdown);
 
 window.Markdown = Markdown;
-window.RMarkdown = RMarkdown;
+window.rMarkdown = RMarkdown;
```

Comment:
@maltheism i'm not sure if this is the PR to incorporate this change because it affects all of Loris, or atleast wherever Markdown is being used. i think changing the Markdown is out of scope. i would agree with laetitia's solution for right now to add the eslint exception comments

Reply:
fair enough! if its limited to this module, i guess that's okay

Reply:
@zaliqarosli that change `window.rMarkdown = RMarkdown;` is just setting the global `window` variable to have "rMarkdown" (Ex. window.rMarkdown). That line removed could still exist in this PR if people are worried about it. Since we would then have window.RMarkdown and window.rMarkdown to use whenever. Although I'm somewhat confident my global search through the code didn't miss anything.

Reply:
Just to add my grain of salt here...  As this file will/should be rewritten soon, I tend to think that it's preferable to add the eslint ignore line and not modifying the core. Once this file will be reactified, the core change will not be necessary anymore, and this change break the naming policy consistency. I agree that it's preferable to avoid using eslint ignore for long term change though :)

Reply:
@laemtl looking at this code in question. I doubt that's anytime soon.